tests/test_api.py

```diff
@@ -14,10 +14,7 @@ def test_no_auth_get(client):
 def test_cmd_test_command(client, basic_auth, mock_post_request, ssh_server, test_command):
     # generate a key pair
     response = client.get(
-        f"/api/v1.0/cmd/{test_command}",
-        json={
-            "ssh_port": ssh_server.port,
-        },
+        f"/api/v1.0/cmd/{test_command}?ssh_port={ssh_server.port}",
```

Comment:
good example ... ssh_server.port should probably not be overridable via api parameters